services/src/northbound/src/main/java/org/openkilda/northbound/utils/NorthboundExceptionHandler.java

```diff
@@ -65,6 +65,11 @@
                 break;
             case OPERATION_TIMED_OUT:
             case INTERNAL_ERROR:
+                status = HttpStatus.INTERNAL_SERVER_ERROR;
+                break;
+            case CREATION_FAILURE:
```

Comment:
#659 requests to change the status code for UnroutablePathException only, but this code changes affect all CREATION_FAILURE errors.